internal/hcsoci/network.go

```diff
@@ -1,28 +1,6 @@
 package hcsoci
 
-import (
-	"github.com/Microsoft/hcsshim/internal/hns"
-	"github.com/sirupsen/logrus"
-)
-
-func createNetworkNamespace(coi *createOptionsInternal, resources *Resources) error {
-	netID, err := hns.CreateNamespace()
-	if err != nil {
-		return err
-	}
-	logrus.Infof("created network namespace %s for %s", netID, coi.ID)
-	resources.netNS = netID
-	resources.createdNetNS = true
```

Comment:
This is only assigned here so we can remove this from `Resources` and remove the `ReleaseResources r.createdNetNS` cleanup.